### DISCUSSION TITLE
Do the examples provided by the official Jupyter notebook only support the version installed via pip install py-data-juicer?

### DISCUSSION THREAD

**Discussion by Tendo33** [datajuicer/data-juicer#475] [General]
I found that many features are missing from the source code installation.
Such as:
```
process_list = executor.process_list

'Executor' object has no attribute 'process_list'
```

**Comment by HYLcool** [comment]
Hi @Tendo33 , thanks for your attention on Data-Juicer!

Sorry for the trouble. The `process_list` and `ops` attributes of `Executor` are removed from the PR #396  months ago because the configs in `process_list` are stored in the `op._op_cfg` attribute in `ops` and there is no need to make `ops` as an attribute of `Executor`.

For regular usage such as running the `dj-process` command, the information in `Executor.process_list` before is visualized in the `process` field of the configuration table in the logs, so you can refer to it.

As for the jupyter notebooks in the KDD tutorials, we will update them ASAP to the latest version of Data-Juicer.

Thanks for your finding!

**Reply by Tendo33** [reply]
Hi, I hope you are always well. May I ask when the next release version of Data Juicier will be available?
